### PR TITLE
Enhancement: Synchronize workflow

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] synchronizes the workflow configuration between `master` and `next`

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ on:
   pull_request:
   push:
     branches:
-      - master
-      - next
+      - "master"
```
Comment (localheinz) [inline .github/workflows/continuous-integration.yml]: Consistently using quotes.

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -15,114 +15,123 @@ jobs:
...
+          - "7.2"
 
     steps:
       - name: "Checkout"
-        uses: actions/checkout@v1
+        uses: "actions/checkout@v1"
 
       - name: "Install PHP with extensions"
-        uses: shivammathur/setup-php@v1
+        uses: "shivammathur/setup-php@v2"
```
Comment (localheinz) [inline .github/workflows/continuous-integration.yml]: Using the latest version here.

File: .github/workflows/continuous-integration.yml [hunk 3]
```diff
@@ -15,114 +15,123 @@ jobs:
...
       - name: "Cache dependencies installed with composer"
-        uses: actions/cache@v1
+        uses: "actions/cache@v1"
         with:
-          path: ~/.composer/cache
-          key: php${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}
-          restore-keys: |
-            php${{ matrix.php-version }}-composer-
+          path: "${{ steps.determine-composer-cache-directory.outputs.directory }}"
+          key: "php-${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}"
```
Comment (localheinz) [inline .github/workflows/continuous-integration.yml]: Slightly modified cache keys.

File: .github/workflows/continuous-integration.yml [hunk 4]
```diff
@@ -15,114 +15,123 @@ jobs:
...
 
       - name: "Cache dependencies installed with composer"
-        uses: actions/cache@v1
+        uses: "actions/cache@v1"
         with:
-          path: ~/.composer/cache
-          key: php${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}
-          restore-keys: |
-            php${{ matrix.php-version }}-composer-
+          path: "${{ steps.determine-composer-cache-directory.outputs.directory }}"
```
Comment (localheinz) [inline .github/workflows/continuous-integration.yml]: Determining the composer cache directory instead of assuming it.

**Comment by pimjansen** [comment]
Thanks @localheinz

**Comment by localheinz** [comment]
Thank you, @pimjansen!

**Pull request merged**